pandas/core/arrays/base.py

```diff
@@ -775,10 +775,18 @@ def convert_values(param):
             res = [op(a, b) for (a, b) in zip(lvalues, rvalues)]
 
             if coerce_to_dtype:
-                try:
-                    res = self._from_sequence(res)
-                except TypeError:
-                    pass
+                if op.__name__ in {'divmod', 'rdivmod'}:
+                    try:
+                        a, b = zip(*res)
```

Comment:
is the zip-star necessary?  If we get here shouldn't `res` just be a 2-tuple?  so `a, b = res`?

Reply:
right now `res` is a list of tuples, so I think we have to split each of those with a `zip(*)`

```python
> /Users/taugspurger/sandbox/pandas/pandas/core/arrays/base.py(781)_binop()
    779                     try:
    780                         import pdb; pdb.set_trace()
--> 781                         a, b = zip(*res)
    782                         res = (self._from_sequence(a),
    783                                self._from_sequence(b))

ipdb> res
[(Decimal('0'), Decimal('1')), (Decimal('1'), Decimal('0')), (Decimal('1'), Decimal('1')), (Decimal('2'), Decimal('0'))]
ipdb> n
> /Users/taugspurger/sandbox/pandas/pandas/core/arrays/base.py(782)_binop()
    780                         import pdb; pdb.set_trace()
    781                         a, b = zip(*res)
--> 782                         res = (self._from_sequence(a),
    783                                self._from_sequence(b))
    784                     except TypeError:

ipdb> p a, b
((Decimal('0'), Decimal('1'), Decimal('1'), Decimal('2')), (Decimal('1'), Decimal('0'), Decimal('1'), Decimal('0')))
```